test/ffmpeg-vaapi/encode/hevc.py

```diff
@@ -88,6 +89,30 @@ def test_r2r(self, case, gop, slices, qp, quality, profile):
     vars(self).setdefault("r2r", 5)
     self.encode()
 
+#vme low delay b cqp
+class cqp_ldb(HEVC8EncoderTest):
+  def init(self, tspec, case, gop, slices, bframes, qp, quality, profile):
+    self.caps = platform.get_caps("vme_lowdelayb", "hevc_8")
+    vars(self).update(tspec[case].copy())
+    vars(self).update(
+      bframes      = bframes,
+      case         = case,
+      gop          = gop,
+      qp           = qp,
+      lowdelayb    = 1,
```

Comment:
If lowdelayb can be 1 or 2, then why not make it a test parameter?

Reply:
Artie, the option b_strategy=1 is lowdelayb, but 2 is not a low delay b (2 is another b strategy, Linjie just to follow use the b_strategy in ffmpeg using), the iHD driver only support vme p frame  and vme low delay b frame. 
but no support for the 2.

Reply:
Ok, so lowdelayb implies a true/false (like lowpower).  We should handle it that way.